MAVProxy/modules/mavproxy_map/mp_elevation.py

```diff
@@ -13,15 +13,20 @@
 
 from MAVProxy.modules.mavproxy_map import srtm
 
+TERRAIN_SERVICES = {
+    "SRTM1"      : ("terrain.ardupilot.org", "SRTM1"),
+    "SRTM3"      : ("terrain.ardupilot.org", "SRTM3")
```

Comment:
a comment saying that SRTM3 is 3 arcsecond, 90m, SRTM1 is 30m would be good

Reply:
Add comments